.github/workflows/deploy.yml

```diff
@@ -38,6 +38,7 @@ jobs:
         run: |
           $SSH "#! /bin/bash -e
             pip install --user --upgrade pyyaml
+            pip install --user --upgrade edx-event-routing-backends
```

Comment:
This is not going to do what you think. It will install the package on the host, but not in the openedx Docker container.

Do you think that the package should be installed by default in the openedx Docker image? If yes, consider one of the following:
a. Add "edx-event-routing-backends" to the edx-platform requirements.
b. Add "edx-event-routing-backends" to the extra requirements in the openedx Dockerfile of tutor (in the [nutmeg](https://github.com/overhangio/tutor/pull/635) branch)

If not, then the requirement should be installed just in this demo platform by adding it to the [`OPENEDX_EXTRA_PIP_REQUIREMENTS`](https://docs.tutor.overhang.io/configuration.html?highlight=openedx_extra_pip_requirements#open-edx-customisation) tutor setting.

In all cases, the package must be pinned to an exact version.